src/NServiceBus.NHibernate/Outbox/OutboxPersister.cs

```diff
@@ -87,7 +91,7 @@ public Task SetAsDispatched(string messageId, ContextBag context)
                 {
                     using (var tx = session.BeginTransaction(IsolationLevel.ReadCommitted))
                     {
-                        var queryString = $"update {typeof(OutboxRecord)} set Dispatched = true, DispatchedAt = :date where MessageId IN ( :messageid, :qualifiedMessageId ) And Dispatched = false";
+                        var queryString = $"update {typeof(OutboxRecord)} set Dispatched = true, DispatchedAt = :date, TransportOperations = NULL where MessageId IN ( :messageid, :qualifiedMessageId ) And Dispatched = false";
```

Comment:
I know this is unchanged from previous, but should we be relying upon Type.ToString() here? For example, if this were made an internal nested class for some reason, it would go from `OutboxRecord` to `ParentClass+OutboxRecord`. Should we change this to use `{typeof(OutboxRecord).Name}` ?

(Probably in many other areas as well.)